src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(1, $width-card);
+  justify-content: center;
+  gap: 0 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    width: $width-card;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    line-height: 300px;
+    text-align: center;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $width-card);
   }
+}
```

Comment:
you can add these styles using scss nesting to the main class for avoiding writing class name many times